src/pybind/mgr/test_orchestrator/module.py

```diff
@@ -179,7 +178,6 @@ def _get_ceph_daemons(self):
 
             # parse daemon ID. Possible options: `-i <id>`, `--id=<id>`, `--id <id>`
             patterns = [r'-i\s(\w+)', r'--id[\s=](\w+)']
-            daemon_id = None
```

Comment:
sure you want to remove this line?

Reply:
It is an unused statement. Correct me if I'm wrong, but if daemon_id is no set inside "for" loop,
the "else" statement raises an error so it makes this line useless.